docker/Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG PYTHON_VERSION
+FROM python:${PYTHON_VERSION}-slim
+
+RUN apt-get update \
+    && apt-get install -y libmemcached-dev \
+                          build-essential \
+                          libsqlite3-mod-spatialite binutils libproj-dev gdal-bin libgdal20 libgeoip1 \
+                          mysql-client \
+                          unzip libaio1 \
+                          gettext \
+                          wget git \
+    && apt-get clean
+
+RUN groupadd -r test && useradd --no-log-init -r -g test test
+
+RUN mkdir /geolite2/ \
+    && cd /geolite2/ \
+    && wget -q http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz \
+        http://geolite.maxmind.com/download/geoip/database/GeoLite2-Country.tar.gz \
+    && (cat *.tar.gz | tar zxvf - --wildcards "*.mmdb" --strip-components=1 -i) \
+    && rm *.tar.gz \
+    && cd /
+
+RUN wget -q https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh -O /bin/wait-for-it.sh \
+    && chmod a+x /bin/wait-for-it.sh
+
+ENV PIP_NO_CACHE_DIR=off
+ENV PYTHONDONTWRITEBYTECODE=1
+RUN pip install --upgrade pip
+
+WORKDIR /django
+
+COPY --chown=test:test ./tests/requirements ./tests/requirements
+RUN for f in ./tests/requirements/*.txt; do pip install -r $f; done && \
+    pip install flake8 flake8-isort selenium
+
+COPY --chown=test:test . .
+
+USER test:test
+
+ENV PYTHONPATH "${PYTHONPATH}:/django/:/django/docker/"
+
+VOLUME /django
```

Comment:
Is it necessary ? docker doesn't ask image permission to mount a volume in a directory. If docker compose says `volumes: [.:/django/]` then it will mount it anyway - whether it has been marked as VOLUME or not in the Dockerfile.

Reply:
It's just metadata about the expected volumes, I'd like to keep it in the image if possible